demo/src/app/components/+datepicker/demos/date-initial-state/date-initial-state.ts

```diff
@@ -0,0 +1,10 @@
+import { Component } from '@angular/core';
+
+@Component({
+  selector: 'demo-datepicker-date-initial-state',
+  templateUrl: './date-initial-state.html'
+})
+export class DemoDatepickerDateInitialStateComponent {
+  bsValue: Date = new Date();
+  bsRangeValue: any = [new Date(2017, 7, 4), new Date(2017, 7, 20)];
```

Comment:
Change the range to a week from today. you can do this by example from min-max demo